packages/map-balloon/README.md

```diff
@@ -78,10 +78,14 @@ farm symbol, and that the class name to include that icon is `.wg-purveyor`.
 * `balloonSize`: number (default: 260). Balloon size
 * `borderRadius`: string (default: 8px). Defines the balloon border radius
 * `pinColor`: string (default: #f1c618). Color to be set to the pin icon
+* `pinHighlightColor`: string (defaults to the `pinColor` value). Color to be set to the highlighted pin
 * `pinSize`: number (default: 40). Size to be set to the pin icon
 * `opacity`: number (default: 1)
 * `captionSize`: number (default: 18), Size of the caption text.
 * `imageTextSize`: number (default: 28), Size of the text overlay on images.
+* `onPinClick`: function. Called when the pin is clicked. If set, changes the pin visual into a round button
+* `onBalloonClick`: function. Called when the balloon is clicked. If set, adds an indicator to the balloon text.
+* `reference`: any. A value that will be passed as first argument for `onPinClick` and `onBalloonClick` calls. This can be handy to identify which pin/balloon was clicked.
```

Comment:
`any` looks very flexible, in practice, what will be normally used here? a string? a dom node? a react node? Are we using this property for more than a type or we are always using with the same type?

Reply:
`reference` is never used by this component. It only stores it an send it back at `onPinClick` and `onBalloonClick`. So the type is irrelevant for this component.

At the webmap we use it to pass the producer id (number), but I don't see why we should restrict this proptype.

Reply:
Hmm, webmap could bind this producer id onto the callback and have it executed without arguments then...

Or we could rename `reference` to `value`, like how radio buttons do. (yeah, maybe what bothers me is the prop name)

Or we could pass the producer id as a `data-something` attribute and have the click handlers forward the click event.

But leaving as it is now is not a problem.